src/test/java/org/apache/commons/csv/perf/PerformanceTest.java

```diff
@@ -85,24 +85,12 @@ private long parse(final Reader reader, final boolean traverseColumns) throws IO
         return recordCount;
     }
 
-    private void println(final String s) {
-        System.out.println(s);
-    }
-
-    private long readAll(final BufferedReader in) throws IOException {
```

Comment:
I disagree here, keep this as is so it can be commented out or changed easily when testing.

Reply:
You got it.